### DISCUSSION TITLE
New year

### DISCUSSION THREAD

**Discussion by noahfraiture** [SnaKyEyeS/ADE-Scheduler#1053] [Q&A] [answered]
hello, how does the year system works ? Is there the need of a commit to have access to year 2024-2025 ? The program for this year is available on ade but not ade scheduler

**Comment by jeertmans** [comment, marked as answer]
Hello, the year has been manually fixed in production, see #1050, and we cannot change that. But there is nothing to do here, because we automatically handle multiple year.
